tests/conftest.py

```diff
@@ -385,3 +386,15 @@ def _(*, exit_code=0, stdout=""):
     git = Git()
     yield git
     assert not git.expected_calls
+
+
+@pytest.fixture
+def output_file():
```

Comment:
please either:
- use the `tmp_path` pytest fixture
- use a fixture that uses it

The idea being that we don't want this to happen in the current repo. Also, you won't have to clean afterwards which make things easier to debug (pytest will leave old folders around for the last 3 invocations before deleting)